### PR TITLE
Add reward-strategy argument to CLI

### PR DESCRIPTION
**Pull request by rshtirmer**

### CODE REVIEW

**Comment by rshtirmer** [comment]
Note: merging these edits w/o merging https://github.com/notadamking/RLTrader/pull/111 will throw lead to import errors in cli.py

File: cli.py [hunk 1]
```diff
@@ -5,16 +5,18 @@
...
 from lib.util.logger import init_logger
 from lib.cli.functions import download_data_async
+from lib.env.reward import BaseRewardStrategy, IncrementalProfit, WeightedUnrealisedProfit
 
 np.warnings.filterwarnings('ignore')
 trader_cli = RLTraderCLI()
 args = trader_cli.get_args()
-
+rewards = {"IncrementalProfit": IncrementalProfit, "WeightedUnrealisedProfit": WeightedUnrealisedProfit}
+rewardStrategy = rewards[args.reward_strategy]()
```
Comment (notadamking) [inline cli.py]: `rewards` and `reward_strategy` belong in `RLTraderCLI`

Comment (rshtirmer) [reply cli.py]: I'm using RLTraderCLI to grab the argument, but I'm not quite sure how to pass a class object in as a cli arg w/o this workaround. Please let me know if you have any ideas. Thanks!

Comment (notadamking) [reply cli.py]: Good point. Though it should be renamed `reward_strategy` to keep with the snake_case of the library. Also, I don't think it should be initialized (i.e. remove the `()` after `rewards[args.reward_strategy]`) as the class is already being initialized in the environment.

Comment (rshtirmer) [reply cli.py]: Sounds good! Thanks Adam :)

File: cli.py [hunk 2]
```diff
@@ -36,7 +38,7 @@ def run_optimize(args, logger):
 
     from lib.RLTrader import RLTrader
 
-    trader = RLTrader(**vars(args), logger=logger)
+    trader = RLTrader(**vars(args), logger=logger, reward_strategy=rewardStrategy)
```
Comment (notadamking) [inline cli.py]: Looks like `rewardStrategy` here should be `reward_strategy`, then we can merge~

**Pull request merged**